client/src/services/course.ts

```diff
@@ -25,6 +26,9 @@ export interface CourseTask {
   descriptionUrl: string | null;
   studentStartDate: string | null;
   studentEndDate: string | null;
+  crossCheckEndDate: string | null;
+  crossCheckStatus: CrossCheckStatus;
+  useJury: boolean;
```

Comment:
I think it's not better naming for react ecosystem. I will expect hook with such name.
Can you rename it in something like `isJuryInvolved` or `isCheckedByJury`?